fix/mejoras-point (#2164)
mercadopago/devsite-docs: guides/mp-point/integration-api/configure-devices-pdv.pt.md

> # Configure seu dispositivo no modo Ponto de Venda

Reviewer: ```suggestion
# Configurar dispositivos
```

Reviewer: Manter a consistência com o nome do menu.

> WARNING
>
> Importante
>

> > Antes de configurar o dispositivo Point no modo Ponto de Venda, você deve acessar sua conta do Mercado Pago e ter criado um [aplicativo](/developers/pt/docs/mp-point/additional-content/your-integrations/dashboard) com **PointdeMercadoPago** como o produto a ser integrado.

Reviewer: ```suggestion
Antes de iniciar a configuração do dispositivo Point em modo Ponto de Venda, é necessário que você acesse sua conta no Mercado Pago e crie um [aplicativo](https://chat.openai.com/developers/pt/docs/mp-point/additional-content/your-integrations/dashboard), selecionando PointdeMercadoPago como o produto a ser integrado. Não se esqueça também de verificar suas [credenciais de produção](https://chat.openai.com/developers/pt/docs/mp-point/additional-content/your-integrations/credentials), para assegurar a gestão adequada da sua integração.
```

>
> Além disso, lembre-se de acessar suas [credenciais de produção](/developers/pt/docs/mp-point/additional-content/your-integrations/credentials) para gerenciar corretamente sua integração.

Para iniciar sua integração com o Mercado Pago Point via API, você deve configurar seu dispositivo no modo Ponto de Venda. Para fazer isso, siga as etapas abaixo.

> ## Crie uma loja e um PDV

Reviewer: ```suggestion
## Crie loja e PDV
```

Para configurar o dispositivo Point no modo Ponto de Venda, você deve associá-lo a uma loja e a um ponto de venda nesta loja.

Se você ainda não criou nenhuma loja ou PDV, pode fazê-lo por meio de nossa API.

Primeiro, você deve criar uma loja usando o endpoint [Criar loja](/developers/pt/reference/stores/_users_user_id_stores/post). Você deve substituir os valores `user_id` e `YOUR_ACCESS_TOKEN` pelos obtidos ao criar seu aplicativo, bem como modificar os parâmetros necessários de acordo com as características do seu negócio.

Em seguida, você deve criar um PDV usando o endpoint [Criar PDV](/developers/pt/reference/pos/_pos/post). Este PDV deve ser associado à loja criada anteriormente, portanto, você deve substituir o parâmetro `external_store_id` pelo obtido durante a criação da loja.

> ## Associe o dispositivo Point à sua conta do Mercado Pago

Reviewer: ```suggestion
## Associar dispositivo Point à sua conta Mercado Pago
```

> Para vincular o dispositivo Point à sua conta do Mercado Pago, você precisa ter o aplicativo do Mercado Pago instalado em seu dispositivo móvel.

Reviewer: ```suggestion
Para vincular seu dispositivo Point à sua conta Mercado Pago, é necessário ter o aplicativo Mercado Pago instalado no seu celular.
```

> Faça login no aplicativo com seu nome de usuário e senha e toque no ícone do QR code para escanear o código que aparece quando você liga o dispositivo Point. Dessa forma, o dispositivo será vinculado à sua conta.

Reviewer: ```suggestion
Abra o aplicativo e faça login usando seu nome de usuário e senha. Em seguida, toque no ícone de QR code e escaneie o código exibido ao ligar o dispositivo Point. Ao fazer isso, você estará vinculando o dispositivo à sua conta de maneira prática e segura.
```

> NOTE
>
> Nota
>

> > Se você for operar em nome de outros vendedores, poderá gerenciar a vinculação com segurança, integrando [OAuth.](/developers/pt/docs/mp-point/additional-content/security/oauth/introduction)

Reviewer 2: ```suggestion
> Se você for operar em nome de outros vendedores, poderá gerenciar a vinculação com segurança, integrando [OAuth](/developers/pt/docs/mp-point/additional-content/security/oauth/introduction).
```

## Configure sua loja e PDV

Depois de vincular seu dispositivo Point à sua conta do Mercado Pago e ter criado a loja e o PDV, você deve preencher os dados comerciais e configurar o PDV para associá-los ao dispositivo.

Para fazer isso, você pode acessar o [site do Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]/stores) e ir para **Seu negócio > Lojas e registros**.

> NOTE
>
> Nota
>

> > Você também pode configurar a loja e o PDV diretamente do dispositivo Point, uma vez que esteja vinculado. O próprio dispositivo irá guiá-lo se você escolher configurá-lo dessa maneira.

Reviewer: ```suggestion
> Uma vez que o dispositivo Point esteja vinculado, é possível configurar sua loja e o PDV diretamente por ele. O dispositivo exibirá orientações passo a passo caso opte por essa configuração.
```

> ## Ative o modo PDV no dispositivo Point

Reviewer: ```suggestion
## Ativar modo PDV no dispositivo Point
```
